tests/test_column_family.rs

```diff
@@ -19,18 +19,32 @@ use pretty_assertions::assert_eq;
 use rocksdb::{ColumnFamilyDescriptor, MergeOperands, Options, DB, DEFAULT_COLUMN_FAMILY_NAME};
 use util::DBPath;
 
-#[test]
-fn test_column_family() {
+fn run_test_column_family(is_multi_threaded: bool) {
```

Comment:
It seems, that the test is very complex and tests more than one case. I'd split it up.